src/applications/vaos/tests/containers/VAOSApp.unit.spec.jsx

```diff
@@ -9,6 +9,7 @@ describe('VAOS <VAOSApp>', () => {
   it('should render login gate and downtime notification', () => {
```

Comment:
Would it be useful to also add a test where `verified = false`?

Reply:
Probably wouldn't hurt.